src/templates/base.njk

```diff
@@ -106,6 +106,68 @@
             </div>
           </div>
           <!-- /MOVR price ticker -->
+          <!-- SAMA price ticker -->
```

Comment:
![image](https://user-images.githubusercontent.com/47224690/208292952-0c4aae71-3b4e-44b7-8744-f3ba8ea451a3.png)

Reply:
Price reflected in final calculation
![image](https://user-images.githubusercontent.com/47224690/208293080-351715d2-4b2d-42f5-8254-afe12e908134.png)